packages/form-js-viewer/src/render/components/form-fields/Textarea.js

```diff
@@ -0,0 +1,89 @@
+import { isArray, isObject } from 'min-dash';
+import { useContext } from 'preact/hooks';
+
+import { FormContext } from '../../context';
+
+import Description from '../Description';
+import Errors from '../Errors';
+import Label from '../Label';
+
+import {
+  formFieldClasses,
+  prefixId
+} from '../Util';
+
+const type = 'textarea';
+
+
+export default function Textarea(props) {
+  const {
+    disabled,
+    errors = [],
+    field,
+    value = ''
+  } = props;
+
+  const {
+    description,
+    id,
+    label,
+    validate = {}
+  } = field;
+
+  const { required } = validate;
+
+  const onInput = ({ target }) => {
+
+    props.onChange({
+      field,
+      value: target.value
+    });
+
+    autoSizeTextarea(target);
+
+  };
+
+  const autoSizeTextarea = (textarea) => {
+
+    textarea.style.height = 'inherit';
+
+    const computed = window.getComputedStyle(textarea);
+
+    const height = parseInt(computed.getPropertyValue('border-top-width'))
+      + parseInt(computed.getPropertyValue('padding-top'))
+      + textarea.scrollHeight
+      + parseInt(computed.getPropertyValue('padding-bottom'))
+      + parseInt(computed.getPropertyValue('border-bottom-width'));
+
+    textarea.style.height = `${height}px`;
+
+  };
+
+  const { formId } = useContext(FormContext);
+
+  return <div class={ formFieldClasses(type, errors) }>
+    <Label
+      id={ prefixId(id, formId) }
+      label={ label }
+      required={ required } />
+    <textarea class="fjs-textarea"
+      disabled={ disabled }
```

Comment:
The auto resize is not working on disabled text areas (the scroll bar is visible).

![image](https://user-images.githubusercontent.com/9433996/199234475-e366fb95-7623-419e-a03f-f89c0e1335dd.png)

Reply:
The same happens when using larger input data for a text area.

![image](https://user-images.githubusercontent.com/9433996/199235702-f93a2052-fb88-45ba-93ca-faa466e4b1c6.png)